site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -93,14 +93,18 @@ Unlike dynamic and session rules, static rules are packaged, installed, and upda
 
 To test rule files, [load your extension unpacked](/docs/extensions/mv3/getstarted/development-basics/#load-unpacked). Errors and warnings about invalid static rules are only displayed for unpacked extensions. Invalid static rules in packed extensions are ignored.
 
-### Enable and disable static rulesets {: #update-enabled-rulesets }
+### Enable and disable static rules and rulesets {: #update-rules-rulesets }
 
-To enable or disable static rulesets, call [`updateEnabledRulesets()`](#method-updateEnabledRulesets). This method takes an [`UpdateRulesetOptions`](#type-UpdateRulesetOptions) object, which contains arrays of IDs of rules to enable or disable. The IDs are defined using the `"id"` key of the `Ruleset` dictionary.
+Both individual static rules and complete static rulesets may be enabled or disabled at runtime. {: #update-enabled-rulesets }
```

Comment:
This was the existing anchor. Yes, this works.

Reply:
TIL!